python/extensions.bzl

```
@@ -33,11 +34,17 @@ def _python_impl(module_ctx):
                 register_coverage_tool = toolchain_attr.configure_coverage_tool,
                 ignore_root_user_error = toolchain_attr.ignore_root_user_error,
             )
-            host_hub_name = toolchain_attr.name + "_host_interpreter"
-            _host_hub(
-                name = host_hub_name,
-                user_repo_prefix = toolchain_attr.name,
-            )
+
+            # We collect all of the toolchain names to create
+            # the INTERPRETER_LABELS map.  This is used
+            # by interpreter_extensions.bzl
+            toolchains.append(toolchain_attr.name)
+
+    if len(toolchains) != 0:
+        hub_repo(
+            name = "pythons_hub",
+            toolchains = toolchains,
+        )
```

The hub has to always be created because use_repo() calls aren't conditional

```suggestion
    hub_repo(
        name = "pythons_hub",
        toolchains = toolchains,
    )
```